operator/node.go

```diff
@@ -27,8 +27,8 @@ import (
 // Options contains options to create the node
 type Options struct {
 	// NetworkName is the network name of this node
-	NetworkName         string `yaml:"Network" env:"NETWORK" env-default:"mainnet" env-description:"Network is the network of this node"`
-	CustomDomainType    string `yaml:"CustomDomainType" env:"CUSTOM_DOMAIN_TYPE" env-default:"" env-description:"Override the SSV domain type. This is used to isolate the node from the rest of the network. Do not set unless you know what you are doing. This would be incremented by 1 for Alan, for example: 0x01020304 becomes 0x01020305 post-fork."`
+	NetworkName         string `yaml:"Network" env:"NETWORK" env-default:"mainnet" env-description:"Ethereum network to connect to (mainnet, goerli, etc.)"`
+	CustomDomainType    string `yaml:"CustomDomainType" env:"CUSTOM_DOMAIN_TYPE" env-default:"" env-description:"Override SSV domain type for network isolation (e.g., 0x01020304 → 0x01020305 post-fork). Only modify if you know what you're doing."`
```

Comment:
We need to keep `This would be incremented by 1 for Alan` because it's an Alan fork workaround, we didn't increment it before the Alan fork. Now we increment everything passed here by 1, e.g. custom domain type `0x12345678` will in fact be converted to `0x12345679`

```suggestion
	CustomDomainType    string `yaml:"CustomDomainType" env:"CUSTOM_DOMAIN_TYPE" env-default:"" env-description:"Override SSV domain type for network isolation. Only modify if you know what you're doing. This would be incremented by 1 after Alan fork (e.g., 0x01020304 → 0x01020305 post-fork). "`
```